pkg/ocihook/ocihook.go

```diff
@@ -79,7 +80,7 @@ func Run(stdin io.Reader, stderr io.Writer, event, dataStore, cniPath, cniNetcon
 	}
 }
 
-func newHandlerOpts(state *specs.State, dataStore, cniPath, cniNetconfPath string) (*handlerOpts, error) {
+func newHandlerOpts(state *specs.State, dataStore, cniPath, cniNetconfPath, pidFilePath string) (*handlerOpts, error) {
```

Comment:
pidFilePath should be obtained from `o.state.Annotations`

Reply:
thank you so much, fixed.